site/en/docs/extensions/mv3/security/index.md

```diff
@@ -162,7 +162,7 @@ Avoid using `eval()` whenever possible to prevent attacks, as `eval()` will exec
 into it, which may be malicious.
 
 ```js
-var xhr = new XMLHttpRequest();
+var xhr = new fetch();
```

Comment:
`XMLHttpRequest()` and `fetch()` have very different surfaces. You'll need to rewrite the the whole sample.

Reply:
+1